src/index.js

```diff
@@ -0,0 +1,3 @@
+/*jshint esversion: 6 */
+import './css/styles.css';
+import projectsContainer from './views/view.js';
```

Comment:
this import is the one that displays the whoole project essentially . I don't know if the situation is different in a mac , i am using windows 10 and if i remove this import the whole project isn't rendered

Reply:
I am also using windows 10. So you can do this `import {projectsContainer} from './todoModal.js';' [Path name need to be fixed] and then `projectsContainer.initializeContainer();`.  Maybe. Because this project is looking really complicated to understand and give you a proper solution. Or whatever solution you found to solve this problem. If not then skip it. Always remember We don't write code only for us. Write code in a way so that it will be also understandable for others. 🙂

Reply:
Things started to get complicated to me by the time that a lot of things should get built when clicking either on project or on 'Save a new todo'.That's why the code became like a spaghetti in the logic.  Ok i will give it a try .If not i should i merge?

Reply:
Nope. Send another request. 🙂

Reply:
Okay , i tried that but it doesnt work . The thing i can do is to take the code from  the views.js and put it to the index.js . All the things that exist in the views.js is the code that has been processed till that point .

Reply:
Then it's okay to leave it as it is. I hope in future you will remake this project better than this time 🙂  when you finished those react-redux projects. It will be more fun to convert one of your existing projects using another cool tech.

Reply:
Ok thanks Dipto!! I would really appreciate it if you could point out some technics in the project but it is ok

Reply:
In the future, If you will use react-redux for this project you will get all the techniques. 🙂

Reply:
Ok great !